src/components/Markdown/Markdown.js

```diff
@@ -3,18 +3,20 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import remark from 'remark'
 import remark2react from 'remark-react'
-import { SafeLink, GU, theme } from '@aragon/ui'
+import { Link, GU, useTheme } from '@aragon/ui'
 
 const Markdown = React.memo(({ text }) => {
+  const theme = useTheme()
+
   return (
-    <Wrapper>
+    <Wrapper theme={theme}>
       {
         /* eslint-disable react/prop-types */
         remark()
           .use(remark2react, {
             remarkReactComponents: {
               a: ({ children, ...props }) => (
-                <StyledLink {...props}>{children}</StyledLink>
+                <Link target="_blank">{children}</Link>
```

Comment:
I don't understand why we need this `external` prop; setting `target` is a very common way (the only one actually) to set a link's target - you could argue it is less verbose but I mean is it the plan to have a custom API for all components and move away from the native API from the elements these render? I'll update as needed.

Reply:
It can be discussed case by case, and I think we are always looking at what DOM APIs are doing before deciding on aragonUI’s, even though future versions of aragonUI might support other targets than the DOM. But in this case, `target=_blank` sounds a lot more like the way it is implemented (using a `<a>`) than the intention behind it.

For example, we had a conversation this week with @dizzypaty about adding a text underline to `Link` when using `external`, and not otherwise. It would then also become a semantic prop, that could be used without a `href`, for semantic and stylistic purposes (e.g. a `onClick` that does something before opening a new tab).